.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: build
+
+on: [push, pull_request]
```

Comment:
`push` could be restricted to known branches (`master`) to prevent double builds on every PR. See also https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/3